docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx

```diff
@@ -102,6 +102,10 @@ import styles from '@site/src/pages/index.module.css';
 Using **run**, setting **--base-path** and specifying **--chain** became mandatory starting with Gemini 3h.
 :::
 
+:::note
+As of the Jun 11, 2024 version a new option is available for fast syncing called Snap Sync. It will only work for the initial sync, but many people report that they are fully synced within hours, not days.  The parameter is **--sync snap**
```

Comment:
```suggestion
As of the Jun 11, 2024 version a new option is available for fast syncing called Snap Sync](../cli-tips.mdx#Snap Sync). It will only work for the initial sync, but many people report that they are fully synced within hours, not days. The parameter is **--sync snap**.
```